client/src/pages/teds/charts/PrctIpccPublicationsForFiveCountries.tsx

```diff
@@ -0,0 +1,227 @@
+    body: {
+      should: [
+        { term: { "ipcc.wg.keyword": list_wg[0] } },
+        { term: { "ipcc.wg.keyword": list_wg[1] } },
+        { term: { "ipcc.wg.keyword": list_wg[2] } },
+        { term: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+      minimum_should_match: 1,
+    },
+    name: "all",
+  };
+
+  const filter1 = {
+    body: {
+      must: [{ match: { "ipcc.wg.keyword": list_wg[0] } }],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Sciences Physique (WG1)",
+  };
+  const filter2 = {
+    body: {
+      must: [{ match: { "ipcc.wg.keyword": list_wg[1] } }],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations (WG2)",
+  };
+  const filter2cross = {
+    body: {
+      must: [{ match: { "ipcc.wg.keyword": list_wg[2] } }],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations - risques locaux (WG2 cross)",
+  };
+  const filter3 = {
+    body: {
+      must: [{ match: { "ipcc.wg.keyword": list_wg[3] } }],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+    },
+    name: "Atténuations (WG3)",
+  };
+  const filter12 = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Sciences Physique et Adaptations",
+  };
+
+  const filter22cross = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+    },
+    name: "Adaptations et Adaptations - risques locaux",
+  };
+  const filter23 = {
+    body: {
+      must: [
+        { match: { "ipcc.wg.keyword": list_wg[1] } },
+        { match: { "ipcc.wg.keyword": list_wg[3] } },
+      ],
+      must_not: [
+        { match: { "ipcc.wg.keyword": list_wg[0] } },
+        { match: { "ipcc.wg.keyword": list_wg[2] } },
+      ],
+    },
+    name: "Adaptation et Atténuation",
+  };
+
+  const filters = [
+    filter1,
+    filter2,
+    filter2cross,
+    filter3,
+    filter12,
+    filter22cross,
+    filter23,
+  ];
+
+  const countries = [
```

Comment:
Countries pourrait être en constantes extérieure afin d'être facilement changeable. Un fichier de paramètre peut etre. Et en code ISO.

Reply:
ouais je vais changer la base elastic pour qu'on puisse faire les agrégations directement sur les countries

Reply:
I might be wrong but OpenAlex returns full name and not iso in case of aggregations